README.md

```diff
@@ -13,3 +13,31 @@ Please open an issue if you have any questions.
 ### Small Logo
 
 [![Logo Small](gotify-logo-small.png)](gotify-logo-small.png)
+
+### Notification Logo
+
+Logo for the notification drawer was made by Choi Wong and has been optimized by
```

Comment:
You haven't created the icon, isn't it based on the gotify icon, or where have you got it?

Reply:
Friend of mine redraw it inspired on the original. Fine with the fact the Gotify organization has the rights. I will improve the text.

Reply:
https://github.com/PanderMusubi/logo/commit/2c0c95135dcf914b8c502b21e1b3f0c2888da2da